src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+public class Ui {
+
+    enum Keywords {
+        mark,
+        unmark,
+        delete,
+        todo,
+        deadline,
+        event
+    }
+
+    /**
+     * A constructor to initialize the Ui Object.
+     */
+    public Ui() {}
+
+    public static String straightLine = "  ----------------------------------------------------------------------------------";
```

Comment:
I think this line exceeds the 120 character per line hard limit.

Reply:
Fixed the issue by removing in a later update :,)